src/ec/packages/ec-component-footer/README.md

```diff
@@ -1,97 +1 @@
 # ECL Twig - EC Footer component
-
-npm package: `@ecl-twig/ec-component-footer`
-
-```shell
-npm install --save @ecl-twig/ec-component-footer
```

Comment:
Point for discussion, as I left the installation instructions part in this README. I guess it's ok to leave them, as a very basic intro, the rest is for the style guide, yes, but this could be useful for npm package in overall.